cmd/ocm/cluster/create/create.go

```diff
@@ -226,9 +226,11 @@ func run(cmd *cobra.Command, argv []string) error {
 	}
 
 	// Send a request to create the cluster:
-	response, err := cmv1Client.Clusters().Add().
-		Body(cluster).
-		Send()
+	request := cmv1Client.Clusters().Add().
+		Body(cluster)
+	arguments.ApplyParameterFlag(request, args.parameter)
```

Comment:
I prefer we dryRun as command-line parameter as I am trying to separate the API-like commands from other commands. Same as in https://github.com/openshift/moactl/blob/master/cmd/create/cluster/cmd.go#L51 for reference.

Reply:
I thought about that too, sounds good.  Should I remove the non-working --parameter & --header from this command then?

Reply:
yes, good idea